shared/src/api/QuestionRouter.ts

```diff
@@ -1,59 +1,53 @@
+import { SingleTranslation } from "@shared/utils/translations"
+import { collection as globalCollection } from "../../../settings/questionsSelection"
 import { Language } from "./Language"
 import { deserializeParameters, missingParameters, Parameters } from "./Parameters"
 import { QuestionGenerator } from "./QuestionGenerator"
 
 /**
- * Objects of the following type describes the routes to all available question
- * generators.
+ * Objects of the following type describe a sequence of question collections.
  */
-export type QuestionRoutes = Array<{
-  path: string
-  generator: QuestionGenerator
+export type QuestionCollection = Array<{
+  slug: string
+  name: SingleTranslation
+  contents: Array<QuestionGenerator>
+  image?: URL
```

Comment:
Do we want to keep the images? If yes, I wouldn't make this optional, so its mandatory for every collection to have it's own image.

Reply:
Currently, they are not displayed. If image is undefined, there is a default image that would be used instead

Reply:
Yes, currently they are not being displayed, I meant looking into the future. But if a default image can be used it should not matter 👍